config-stage.template

```diff
@@ -0,0 +1,148 @@
+%% -*- erlang -*-
```

Comment:
This is mostly a copy of config.template. see diff between commit 1 and 2 to see what all has changed